### PR TITLE
Missing log variable

### PR DESCRIPTION
**Pull request by jmuheim**
Hi!  This looks like it will be a great gem.

I get an error saying that the log variable in the runner file is undefined.  Perhaps I am not using it as intended - This app is on rails 3.2 still, and most of our tests are test unit, rather than rspec.

And it would be nice if you would add tags in git for the released gem versions.

### CODE REVIEW

**Comment by nruth** [comment]
I also get this error with Rails 4.

```
 HEADHUNTER=true bin/rspec spec/features
/usr/local/var/rbenv/versions/1.9.3-p448/lib/ruby/gems/1.9.1/gems/headhunter-0.1.3/lib/headhunter/runner.rb:44:in `precompile_assets!': undefined local variable or method `log' for #<Headhunter::Runner:0x007fb89dedfa28
```

I've tried running precompile first and that works alone, but still get the above error afterwards.

**Comment by jmuheim** [comment]
I'm in the process of fixing this issue at the moment. I'll get back to you. :+1:

**Comment by jmuheim** [comment]
Hey guys, I removed the `log` variable. I'd like to hear if it's working for you now. :+1:

File: lib/headhunter/runner.rb [hunk 1]
```diff
@@ -44,13 +44,13 @@ def precompile_assets!
       log.print "Headhunter is removing eventually existing assets...".yellow
```
Comment (nruth) [inline lib/headhunter/runner.rb:44]: it still fails here for me; I'm going to fork and try removing the remaining references and see what happens

Comment (jmuheim) [inline lib/headhunter/runner.rb:44]: Damn, how could I miss this? Any PR's are highly appreciated. :+1:

**Pull request merged**